### PR TITLE
Pushing Up Homework #10

### PR DESCRIPTION
**Pull request by snarvasa**
I was a bit unclear on the assignment, but this is what I did:

- I created a new rails app called animal_app
- I used scaffold to create all of the necessary stuff for the animal model, controller, view, etc.
- I created a new table and defined a few columns with their object types
- Updated the gemfile to include all of the debugging tools we went over in class

### CODE REVIEW

**Comment by snarvasa** [comment]
Errr. Sorrry, I pushed this homework up before Travis mentioned we need to check it into a separate branch. My bad!

File: SeanNarvasa-snarvasa/hw_10/animal_app/config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Rails.application.routes.draw do
+  resources :animals
+  # The priority is based upon order of creation: first created -> highest priority.
```
Comment (chylaudes) [inline SeanNarvasa-snarvasa/hw_10/animal_app/config/routes.rb]: Might be helpful to put in
`root 'animal#index'`
that way when you go to localhost:3000/ it goes straight to the animal#index view.

File: SeanNarvasa-snarvasa/hw_10/animal_app/db/migrate/20150902025338_create_animals.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateAnimals < ActiveRecord::Migration
+  def change
+    create_table :animals do |t|
+      t.string :animal_type
+      t.boolean :neuter_status
+      t.string :name
```
Comment (chylaudes) [inline SeanNarvasa-snarvasa/hw_10/animal_app/db/migrate/20150902025338_create_animals.rb]: Loool neuter_status

File: SeanNarvasa-snarvasa/hw_7/lib/apartment.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    @price = nil
+  end
+
+  def to_s #overrides the inherited to_s method, since it is defined within the class.
+    #Anything 'puts''d
+    "Apartment #{number} has #{sqft} square feet. There are #{bedrooms} bedrooms and #{bathrooms} bathrooms."
+  end
+
+  def not_occupied? #This is an instance method.
+    renter.nil?
```
Comment (chylaudes) [inline SeanNarvasa-snarvasa/hw_7/lib/apartment.rb]: This is excellent! Way the go!

**Comment by chylaudes** [comment]
Looks good, great notes and I just put in some comments that will improve your code.  I think you're understanding the Ruby concepts effectively.

**Pull request merged**